nRFCloud/devzone-oauth
# Update authorization and refresh lambdas

* Make two folders: one for DevZone authorization, one for refreshing access token
* Update refresh lambda

**Testing**
1) Login into https://devzone.nordicsemi.com/api.ashx/v2/oauth/authorize?response_type=code&client_id=08bf63d0-483a-4891-a311-9f12a80747bf&redirect_uri=https://local.account.nrfcloud.com/web/
2) The redirect will include a token and refresh token
3) Make a call to https://local.account.nrfcloud.com/web/refresh/?refreshToken=refreshToken
4) The return result should include the refresh token with new token
